src/coord/Weather.cc

```diff
@@ -0,0 +1,72 @@
+// -*- lsst-c++ -*-
+
+/*
+ * LSST Data Management System
+ * Copyright 2008, 2009, 2010 LSST Corporation.
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <http://www.lsstcorp.org/LegalNotices/>.
+ */
+
+/**
+ * @file
+ * @brief Provide functions to handle dates
+ * @ingroup afw
+ * @author Steve Bickerton
+ *
+ */
+#include <sstream>
+#include <cmath>
+
+#include "lsst/pex/exceptions.h"
+#include "boost/format.hpp"
+#include "boost/algorithm/string.hpp"
+#include "boost/tuple/tuple.hpp"
+#include "lsst/afw/coord/Weather.h"
+
+namespace lsst { namespace afw { namespace coord {
+
+Weather::Weather(double airTemperature, double airPressure, double humidity) :
+    _airTemperature(airTemperature),
+    _airPressure(airPressure),
+    _humidity(humidity)
+{
+    validate();
+}
+
+bool Weather::operator==(Weather const & other) const {
+    return (_airTemperature == other.getAirTemperature()
+            && _airPressure == other.getAirPressure()
+            && _humidity == other.getHumidity());
```

Comment:
I'm with @r-owen on this.  I've never understood the preference for break after operator.  Putting the operator on the next line makes it so much more like the way we write out mathematics.

As for safety, isn't that a moot point in a case where you've got parentheses on both sides?

All that said, the coding standards do *suggest* break after operator, though the implication is this is one of several considerations in making an expression easier to read (and I feel it almost never does).